api/src/main/java/me/lokka30/treasury/api/economy/misc/EconomyAPIVersion.java

```diff
@@ -12,14 +12,55 @@
 
 package me.lokka30.treasury.api.economy.misc;
 
-@SuppressWarnings("unused")
+/**
+ * Represents the current API version.
+ *
+ * @author lokka30, MrIvanPlays
+ * @since v1.0
+ */
 public enum EconomyAPIVersion {
+    v1_0(new short[] {1, 0})
+    ;
+
+    /**
+     * Returns the latest economy api version.
+     *
+     * @return latest economy api version
+     */
+    public static EconomyAPIVersion getLatestVersion() {
```

Comment:
I believe `getCurrentVersion()` is a better name, because the user could have a very old version of Treasury installed so an economy provider could think that Treasury is providing the latest one available online.